test/DSROracle.t.sol

```diff
@@ -0,0 +1,189 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import "forge-std/Test.sol";
+
+import { PotMock } from "./mocks/PotMock.sol";
+
+import { DSROracle } from "../src/DSROracle.sol";
+
+contract DSROracleTest is Test {
+
+    uint256 constant FIVE_PCT_APY_DSR = 1.000000001547125957863212448e27;
+    uint256 constant FIVE_PCT_APY_APR = 0.048790164207174267760128000e27;
+
+    PotMock   pot;
+    DSROracle oracle;
+
+    function setUp() public {
+        skip(30 * (365 days));  // Skip 30 years to avoid underflow
+
+        pot = new PotMock();
+        oracle = new DSROracle(address(pot));
+    }
```

Comment:
```suggestion
    uint256 YEAR;

    PotMock   pot;
    DSROracle oracle;

    function setUp() public {
        skip(30 * (365 days));  // Skip 30 years to avoid underflow

        YEAR = block.timestamp + 365 days;

        pot    = new PotMock();
        oracle = new DSROracle(address(pot));
    }
```

Reply:
And then replace `block.timestamp + 365 days` everywhere and align all asserts